shared/src/business/useCases/externalDocument/uploadExternalDocumentsInteractor.js

```diff
@@ -86,24 +94,33 @@ exports.uploadExternalDocumentsInteractor = async ({
       ].docketEntryId = await uploadDocumentAndMakeSafeInteractor(
         `secondarySupporting${i}`,
       );
+      docketEntryIdsAdded.push(
+        documentMetadata.secondarySupportingDocuments[i].docketEntryId,
+      );
     }
   }
 
   if (leadDocketNumber) {
-    return await applicationContext
-      .getUseCases()
-      .fileExternalDocumentForConsolidatedInteractor({
-        applicationContext,
-        docketNumbersForFiling,
-        documentMetadata,
-        leadDocketNumber,
-      });
+    return {
+      caseDetail: await applicationContext
+        .getUseCases()
+        .fileExternalDocumentForConsolidatedInteractor({
+          applicationContext,
+          docketNumbersForFiling,
+          documentMetadata,
+          leadDocketNumber,
+        }),
+      docketEntryIdsAdded,
+    };
   } else {
-    return await applicationContext
-      .getUseCases()
-      .fileExternalDocumentInteractor({
-        applicationContext,
-        documentMetadata,
-      });
+    return {
+      caseDetail: await applicationContext
+        .getUseCases()
+        .fileExternalDocumentInteractor({
+          applicationContext,
+          documentMetadata,
+        }),
+      docketEntryIdsAdded,
```

Comment:
Looks like we're not testing anything for this new return value (or caseDetail for that matter) in our test coverage for this file. Can we address that?

Reply:
Test has been updated to review the structure of the return value.

Reply:
Thank you!